easysid.cpp

```diff
@@ -65,10 +65,10 @@ EXPORT void set_pw(uint8_t voice, uint16_t pw)
 
 EXPORT void set_wave(uint8_t voice, uint8_t wave)
 {
-    if (voice < 1 || voice > 3) {
+    if (voice < 0 || voice > 2) {
         return;
     }
-    if (wave >= 4) {
+    if (wave > 3) {
```

Comment:
This was done to match the rest of the file, and the range checking it is doing.  This just makes everything more consistent.